safe-global/safe-wallet-web
# fix: Revert wallet balance polling

## What it solves

Reverts #3011 as it was causing too many RPC requests (An additional `eth_chainId` every 15s)
Reverts #2983 as it was causing too many RPC requests (From 8 -> 14 `eth_chainId` requests on initial load)

## How to test it

- Check that the last connected wallet still automatically reconnects
- Check that the wallet balance is visible on initial load

1. Open Safe
2. Open the network tab
3. Observe that there are no additional `eth_chainId` requests over time
